### DISCUSSION TITLE
Add Zwavejs2mqtt as secondary controller

### DISCUSSION THREAD

**Discussion by mgf909** [zwave-js/zwave-js-ui#1125] [Q&A] [answered]
I have an existing stable and functional zwave network running on Openhab2. Im interested in looking into Zwavejs2mqtt and wondering if i can add a second zwave stick to my existing network and operate it with Zwavejs2mqtt as a secondary controller?

This would allow me to take my time to understand Zwavejs2mqtt and allow me to test Openhab3 and HomeAssistant using Zwavejs2mqtt as the gateway....

**Comment by robertsLando** [comment]
I'm not 100% sure if zwave-js will work with this setup. @AlCalzone ?

**Comment by AlCalzone** [comment, marked as answer]
That's not possible at the moment. Zwave-js expects to be the primary controller.

**Reply by mgf909** [reply]
@AlCalzone - OK thanks - If this isnt on the roadmap could it be added? I think it could help people transition across gradually and help protect the hard fought WAF!

Regardless, greatly appreciate your efforts! I'll still have a play but build a separate test z-wave network...
